cmd/fs-v1-metadata.go

```diff
@@ -274,167 +261,134 @@ func newFSMetaV1() (fsMeta fsMetaV1) {
 	return fsMeta
 }
 
-// newFSFormatV2 - initializes new formatConfigV1 with FS format version 2.
-func newFSFormatV2() (format *formatConfigV1) {
+// newFSFormatV1 - initializes new formatConfigV1 with FS format info.
+func newFSFormatV1() (format *formatConfigV1) {
 	return &formatConfigV1{
-		Version: "1",
-		Format:  "fs",
+		Version: formatFileCurrent,
+		Format:  formatBackendFS,
 		FS: &fsFormat{
-			Version: fsFormatV2,
+			Version: fsFormatBackendCurrent,
 		},
 	}
 }
 
-// Checks if input format is version 1 and 2.
-func isFSValidFormat(formatCfg *formatConfigV1) bool {
-	// Supported format versions.
-	var supportedFormatVersions = []string{
-		fsFormatV1,
-		fsFormatV2,
-		// New supported versions here.
-	}
-
-	// Check for supported format versions.
-	for _, version := range supportedFormatVersions {
-		if formatCfg.FS.Version == version {
-			return true
-		}
-	}
-	return false
-}
-
-// errFSFormatOld- old fs format.
-var errFSFormatOld = errors.New("old FS format found")
-
 // Checks if the loaded `format.json` is valid and
 // is expected to be of the requested version.
-func checkFormatFS(format *formatConfigV1, formatVersion string) error {
+func checkFormatFS(format *formatConfigV1) error {
 	if format == nil {
 		return errUnexpected
 	}
 
-	// Validate if we have the same format.
-	if format.Format != "fs" {
-		return fmt.Errorf("Unable to recognize backend format, Disk is not in FS format. %s", format.Format)
+	if format.FS == nil {
+		return errUnexpected
 	}
 
-	// Check if format is currently supported.
-	if !isFSValidFormat(format) {
-		return errCorruptedFormat
+	// Validate if format config version is v1.
+	if format.Version != formatFileCurrent {
```

Comment:
As this function accepts `formatConfigV1`, the expected version should be `formatFileV1`